src/components/auth/ForgotPassword.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from "react"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['useEffect' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5684995)